Userland/Libraries/LibJS/CyclicModule.cpp

```diff
@@ -191,10 +191,23 @@ ThrowCompletionOr<Promise*> CyclicModule::evaluate(VM& vm)
     // 2. Assert: module.[[Status]] is linked, evaluating-async, or evaluated.
     VERIFY(m_status == ModuleStatus::Linked || m_status == ModuleStatus::EvaluatingAsync || m_status == ModuleStatus::Evaluated);
 
+    // NOTE: The spec does not catch the case where evaluate is called twice on a script which failed
+    //       during evaluation. This means the script is evaluated but does not have a cycle root.
+    //       In that case we first check if this module itself has a top level capability.
+    //       See also: https://github.com/tc39/ecma262/issues/2823 .
+
+    // If module.[[TopLevelCapability]] is not empty, then
```

Comment:
Missing step number

Reply:
This (currently) has no spec number, should I just put `?.` for now? Or leave out the spec comments

Reply:
if it has a comment, it must have a number :)
Either give it the same as where you copied it from, or omit - up to you